packages/frontend/src/hooks/useTransactionHistory.ts

```diff
@@ -84,14 +86,14 @@ export const useTransactionHistory = () => {
     },
   )
 
-  const transactions = (data?.swaps || []).map((s: transactions_swaps) => {
+  const transactions = (swaps || []).map((s) => {
     const squeethAmount = new BigNumber(isWethToken0 ? s.amount1 : s.amount0)
     const ethAmount = new BigNumber(isWethToken0 ? s.amount0 : s.amount1)
     const time = new Date(Number(s.timestamp) * 1000).setUTCHours(0, 0, 0) / 1000
     const usdValue = ethAmount.multipliedBy(ethPriceMap[time]).abs()
 
     let transactionType = TransactionType.BUY
-    if (squeethAmount.isPositive() && s.recipient.toLowerCase() === swapRouter.toLowerCase()) {
+    if (squeethAmount.isPositive() && s.recipient.toLowerCase() !== shortHelper.toLowerCase()) {
```

Comment:
hm not sure I understand this, we also want to include txns that go through shortHelper right? Or are we not tracking them here because we are tracking them elsewhere and this is just meant to be for swapRouter txns?

Reply:
There are two conditions here, if s.recipient.toLowerCase() === shortHelper.toLowerCase(), that means users are doing OPEN_SHORT or CLOSE_SHORT, and if s.recipient.toLowerCase() !== shortHelper.toLowerCase(), that means users are doing BUY or SELL, but the pattern wont work for minted/burnt only txs, will figure out a way to distinguish them in the following pr.

Reply:
Ok gotcha so this only affects the txn history, not the positions stuff, and it's just showing the distinction between open_short vs. minting vs. selling? 

And the next PR you're adding the separate actions?

Reply:
Yh after this pr I will add separate actions, cuz it requires new vault history to distinguish minting from MINT_SHORT